plugins/wasi_ocr/wasiocrfunc.h

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Second State INC
+
+#pragma once
+
+#include "runtime/callingframe.h"
+#include "wasiocrbase.h"
+
+#include <cstdint>
+
+namespace WasmEdge {
+namespace Host {
+
+class WasiOCRNumOfExtractions : public WasiOCR<WasiOCRNumOfExtractions> {
+public:
+  WasiOCRNumOfExtractions(WASIOCR::WasiOCREnvironment &HostEnv)
+      : WasiOCR(HostEnv) {}
+  Expect<uint32_t>
+  body(const Runtime::CallingFrame &, uint32_t ImagePathPtr,
+       uint32_t ImagePathLen); //, uint32_t NumOfExtractionsPtr);
```

Comment:
What does this comment mean?

Reply:
I was originally planning on having three arguments, with a pointer to the value needed, but later changed the function to return the value instead, will remove the unused code I had commented.